cmd/minikube/cmd/dashboard.go

```diff
@@ -128,7 +129,12 @@ var dashboardCmd = &cobra.Command{
 			exit.WithCodeT(exit.Unavailable, "{{.url}} is not accessible: {{.error}}", out.V{"url": url, "error": err})
 		}
 
-		if dashboardURLMode {
+		//check if current user is root
+		user, err := user.Current()
+		if err != nil {
+			exit.WithError("Unable to get current user", err)
+		}
+		if dashboardURLMode && user.Uid == "0" {
```

Comment:
I think this should be a `||` instead of `&&`, and expect . that this might fail the integration tests as is.

Reply:
Thanks @tstromberg for the update. I think I focussed too much on root , will update the commit with ||